File: challenges/codeeval/easy/024/victor.ayalamarin.cs
using System;
using System.IO;

namespace ConsoleApplication1
{
    class Program
    {
        static void Main(string[] args)
        {
            int suma = 0;
            string line;
            StreamReader file = new StreamReader("C:\\Users\\VICTOR\\Documents\\Aprueba.txt");
            ///using (StreamReader file = File.OpenText(args[0]))
                do
                {
                line = file.ReadLine();
                if (line != null)
                {
                    
                    line = line.Trim();
                    int aux = 0;
                    aux = Int16.Parse(line);
                    suma = suma + aux;
                }

            } while (line!= null);
            Console.WriteLine(suma);
        }
    }
}

File: challenges/codeeval/easy/091/victor.ayalamarin.java

package codeeval;

import java.io.File;
import java.io.IOException;
import java.text.DecimalFormat;
import java.util.Arrays;
import java.util.Scanner;
import java.util.Stack;

/**
 *
 * @author VICTOR
 */
public class SimpleSorting {
    public static void main(String[] args) throws IOException {
        String linea;
        String ruta="C:\\Users\\VICTOR\\Documents\\Aprueba.txt";
        File file = new File(ruta);
        //File file = new File(args[0]);
        try (Scanner leeArc = new Scanner(file)) {
           while (leeArc.hasNext()){
                linea=(leeArc.nextLine());
                Scanner leeLin = new Scanner(linea);
                Stack numeros =new Stack();
                do {
                    numeros.push(Double.parseDouble(leeLin.next()));
                }while(leeLin.hasNext());
                int tam=numeros.size();
                double num[]= new double[tam];
                for (int i=0;i<tam;i++){
                    num[i]=(double) numeros.pop();
                }
                Arrays.sort(num);
                String respu="";
                DecimalFormat df = new DecimalFormat("0.000");
                for (int j=0;j<tam;j++){
                    double aux;
                    respu=respu+" "+df.format(num[j]);
                 }
                System.out.println(respu.trim());
            }
            leeArc.close();  
        } 
    }
}
